velox/exec/HashJoinBridge.h

```diff
@@ -115,6 +152,9 @@ class HashJoinBridge : public JoinBridge {
  private:
   uint32_t numBuilders_{0};
 
+  // The result of the build side of the current processing unit. There could be
```

Comment:
The comment ins confusing. Can we remove? Not sure what's processing unit mean here.